website/docs/plugins/rq_launcher.md

```diff
@@ -39,25 +39,27 @@ The default configuration is as follows:
 # @package hydra.launcher
 _target_: hydra_plugins.hydra_rq_launcher.rq_launcher.RQLauncher
 enqueue:
-  job_timeout: null                  # maximum runtime of the job before it's killed (e.g. "1d" for 1 day, units: d/h/m/s), default: no limit
-  ttl: null                          # maximum queued time before the job before is discarded (e.g. "1d" for 1 day, units: d/h/m/s), default: no limit
-  result_ttl: null                   # how long successful jobs and their results are kept (e.g. "1d" for 1 day, units: d/h/m/s), default: no limit
-  failure_ttl: null                  # specifies how long failed jobs are kept (e.g. "1d" for 1 day, units: d/h/m/s), default: no limit
-  at_front: false                    # place job at the front of the queue, instead of the back
-  job_id: null                       # job id, will be overidden automatically by a uuid unless specified explicitly
-  description: null                  # description, will be overidden automatically unless specified explicitly
-queue: default                       # queue name
+  job_timeout: null                               # maximum runtime of the job before it's killed (e.g. "1d" for 1 day, units: d/h/m/s), default: no limit
```

Comment:
Can you remove the comments? 
This feels like it's too expensive to maintain. There is already a link to the config that contains the comment (maybe make it more obvious that the link is a place to find the documentation about the fields).

Reply:
Removed the comments ✅. I also slightly rephrased the link to the `config.py` file, to make it more explicit that comments can be found there:

> ~~"The configuration packaged with the plugin is defined [here]."~~

> "Further descriptions on the variables can be found in the plugin config file, defined [here]."

See https://github.com/facebookresearch/hydra/pull/1588/commits/f8780eaf765128aa5e5795868554a34ae19670a6